config/app.php

```diff
@@ -169,10 +169,10 @@
         /*
          * Application Service Providers...
          */
-        App\Providers\AppServiceProvider::class,
-        App\Providers\AuthServiceProvider::class,
+        // App\Providers\AppServiceProvider::class,
```

Comment:
let's not comment, but remove. We can always fallback to specific commit if necessary